packages/providers/src.ts/base-provider.ts

```diff
@@ -614,9 +614,10 @@ export class BaseProvider extends Provider {
         if (params.filter.topics && params.filter.topics.length > 0) {
             for(let i=0; i< params.filter.topics.length; i++) {
                 const topic = params.filter.topics[i];
```

Comment:
Filtering by topicId requires a timestamp range to be set, otherwise the request would fail with status 400